docs/schema.md

```diff
@@ -0,0 +1,75 @@
+
+In fact, earlier releases of the toolkit only generated and supported Web Mercator (EPSG:3857) pyramids, both to minimize GET requests and avoid reprojection on the client. Release `v3.0.0` of `@carbonplan/maps` and `v0.1.0` of `ndpyramid` added support for the Equidistant Cylindrical (EPSG:4326) projection for cases in which users want pyramids in the same projection as the original data, at the [expense of slower rendering times](https://nasa-impact.github.io/zarr-visualization-report/approaches/dynamic-client/e2e-results-projection.html).
+
+## Pyramid schema
+
+While the [map zoom level quadtree structure](https://docs.mapbox.com/help/glossary/zoom-level/#zoom-level-quadtrees) has been used for many years, there was no convention for storing the quadtree pyramids in Xarray and Zarr when we started work on this toolkit (although parallel development occurred in the microscopy and other communities). Therefore, we created a pyramid and metadata schema for `ndpyramid`. The resulting Zarr store for a dataset with one `tavg` data variable would look like:
+
+```{code}
+/
+ ├── .zmetadata
+ ├── 0
+ │   ├── tavg
+ │       └── 0.0
+ ├── 1
+ │   ├── tavg
+ │       └── 0.0
+ │       └── 0.1
+ │       └── 1.0
+ │       └── 1.1
+ ├── 2
+...
+```
+
+Note the quadrupling of the number of chunks as zoom level increases. This, combined with the global extent of individual levels and specific projection, allows inference of the placement of chunks on a web map based on the chunk index.
+
+Metadata about the pyramids is stored in the `multiscales` attribute of the Xarray DataTree or Zarr store:
+
+```{code}
+{
+  "multiscales": [
+    {
+      "datasets": [
+        {
+          "path": "0",
+          "pixels_per_tile": 128,
+          "crs": "EPSG:3857"
+        },
+        {
+          "path": "1",
+          "pixels_per_tile": 128,
+          "crs": "EPSG:3857"
+        }
+        ...
+      ],
+      "metadata": {
+        "args": [],
+        "method": "pyramid_reproject",
+        "version": "0.0.post64"
+      },
+      "type": "reduce"
+    }
+  ]
+}
+```
+
+If the `"crs"` attribute is not included, the pyramid levels are assumed to be in the Web Mercator projection.
+
+In addition, the mapping toolkit relies on the `_ARRAY_DIMENSIONS` attribute introduced by Xarray, which stores the dimension names.
+
+## Pyramids for @carbonplan/maps
+
+In addition to following the quadtree pyramid structure and metadata schema, the pyramids currently must also meet the following requirements for use with `@carbonplan/maps`:
+
+- Consistent chunk size across pyramid levels (128, 256, or 512 are recommended)
+- [zlib](https://numcodecs.readthedocs.io/en/stable/zlib.html) or [gzip](https://numcodecs.readthedocs.io/en/stable/gzip.html) compression
+- Web Mercator (EPSG:3857) or Equidistant Cylindrical (EPSG:4326) projection
```

Comment:
Should we add a bullet about the support `dtype`s? Confusingly, they vary slightly across [Zarr v2](https://github.com/freeman-lab/zarr-js/blob/5179eea3f23bd8e42af004258f4fc7aa018e6018/src/v2.js#L309-L322) and [Zarr v3](https://github.com/freeman-lab/zarr-js/blob/5179eea3f23bd8e42af004258f4fc7aa018e6018/src/v3.js#L283-L289).

Reply:
Good point, thanks! Since V3 is experimental, I'll just reference the V2 dtypes